draft-packages/popover/KaizenDraft/Popover/styles.scss

```diff
@@ -136,6 +146,10 @@ $large: 450px;
   color: $kz-color-cluny-500;
 }
 
+.guidanceIcon {
+  color: $kz-color-cluny-500;
+}
+
```

Comment:
Same as above for Icon. Others may disagree with this thinking but assuming they should be the same, I think we should make that as obvious as possible in the code.